docs/usage.md

```diff
@@ -80,27 +80,29 @@ Output from Variant Calling and/or Annotation will be in a specific directory fo
+| `spring_2` | Full path to spring-compressed, gzipped FastQ file for read 2. The Fastq file has to be first gzipped, then spring-compressed, and it must have the extension `.fastq.gz` or `.fq.gz`.                                                                                                                                            |
+| `bam`      | Full path to (u)BAM file                                                                                                                                                                                                                                                                                                          |
+| `bai`      | Full path to BAM index file                                                                                                                                                                                                                                                                                                       |
+| `cram`     | Full path to CRAM file                                                                                                                                                                                                                                                                                                            |
+| `crai`     | Full path to CRAM index file                                                                                                                                                                                                                                                                                                      |
+| `table`    | Full path to recalibration table file                                                                                                                                                                                                                                                                                             |
+| `vcf`      | Full path to vcf file                                                                                                                                                                                                                                                                                                             |
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
 ### Start with mapping (`--step mapping` [default])
 
-This step can be started either from FastQ files or (u)BAMs. The CSV must contain at least the columns `patient`, `sample`, `lane`, and either `fastq_1/fastq_2` or `bam`.
+This step can be started either from FastQ files (gzip-compressed or gzip+spring-compressed) or (u)BAMs. The CSV must contain at least the columns `patient`, `sample`, `lane`, and `fastq_1/fastq_2`, `spring_1/spring_2` or `bam`.
```

Comment:
```suggestion
This step can be started either from FastQ files (gzip-compressed or gzip+spring-compressed) or (u)BAMs. The CSV must contain at least the columns `patient`, `sample`, `lane`, and either `fastq_1/fastq_2`, `spring_1`, `spring_1/spring_2` or `bam`.
```

Reply:
Adding `spring_1` is fine, but I'm not sure we should add the word `either`.

Here you see an input-csv with contains `fastq_1`, `fastq_2`, `spring_1`, `spring_2`, `bam` and `bai`:

https://github.com/asp8200/sarek/blob/spring_II/tests/csv/3.0/bam_and_fastq_and_spring.csv

One row with `fastq_1/fastq_2` set, one row with `spring_1`, one row with `spring_1` and `spring_2` and one row with `bam` and `bai`.

Reply:
oh I see